### DISCUSSION TITLE
Single threaded frame limiter higher than expected

### DISCUSSION THREAD

**Discussion by Flamiii** [ppy/osu#20152] [Q&A]
Refresh rate - 144hz

What it currently is:
VSync - 144hz (displays as 576hz even though it's capped at 144hz)
2x refresh rate - 576hz
4x refresh rate - 1000hz
8x refresh rate - 1000hz
Basically unlimited - 1000hz

What I expected it to be (identical to multi threaded draw fps):
VSync - 144hz
2x refresh rate - 288hz
4x refresh rate - 576hz
8x refresh rate - 1000hz
Basically unlimited - 1000hz

Personally, I feel that the latter makes way more sense and gives me more precision to set the frame limiter to a value I actually want.

**Comment by peppy** [comment]
I think this is already being tracked somewhere. It needs to match stable really. All the "x refresh rate" will go away.
